models/h2o_dai_german_credit/app_h2o_python_scoring.py

```diff
@@ -0,0 +1,139 @@
+from certifai.model.sdk import SimpleModelWrapper
+import pandas as pd
+import numpy as np
+from numpy import nan
+import datatable as dt
+from scipy.special._ufuncs import expit
+from scoring_h2oai_experiment_7fa37640_0c78_11eb_9e3d_0242ac110003 import Scorer
```

Comment:
The above import is specific to the downloaded pipeline. We need to figure out how to handle this from a "drop in" perspective and for the user who is trying to follow this example.

Reply:
Can use https://docs.python.org/3/library/importlib.html and ask the user to provide the name of the scoring `whl` directly